pubsub/unit_tests/test_topic.py

```diff
@@ -779,10 +779,35 @@ def test_context_mgr_failure(self):
         self.assertEqual(list(batch.messages), [MESSAGE1, MESSAGE2])
         self.assertEqual(getattr(api, '_topic_published', self), self)
 
+    def test_batch_messages(self):
```

Comment:
This unit test was added because my _previous_ unit tests (now corrected) were accepting invalid data but still passing.